src/tools/native-client/src/main/java/com/vesoft/client/NativeClientResourceLoader.java

```diff
@@ -1,47 +0,0 @@
-/* Copyright (c) 2018 vesoft inc. All rights reserved.
- *
```

Comment:
why remove `NativeClientResourceLoader`, maybe use `NativeClientResourceLoader.resourceLoader();` could load `.so`

Reply:
We need to use it in nebula-java, but not here? We just need to pack the `so` in jar in codec.